### DISCUSSION TITLE
export buttons at the bottom of the upsetjs() plot

### DISCUSSION THREAD

**Discussion by baeywen** [upsetjs/upsetjs#87] [Q&A] [answered]
Question1: I saw the share button at the bottom of the plot

![image](https://user-images.githubusercontent.com/50840241/134626957-a958c8c0-bbaf-4706-a3ab-a939c1a577fd.png)

The button links to https://upset.js.org/app/embed.html? , Does that mean each time we use upsetjs(), it will create a copy saved to https://upset.js.org/app/embed.html?

Question 2: Is there any way to just show png and svg button?

**Comment by sgratzl** [comment, marked as answer]
> Does that mean each time we use upsetjs(), it will create a copy saved to https://upset.js.org/app/embed.html?

not nothing will ever be saved on upset.js.org. When you click on the share button you a very long URL will be generated that encodes all your data in the URL and is shown using https://upset.js.org/app/embed.html

The default option in R is only to en/disable the buttons. However, the library supports your request

```R
listInput <- list(one = c(1, 2, 3, 5, 7, 8, 11, 12, 13), two = c(1, 2, 4, 5, 10), three = c(1, 5, 6, 7, 8, 9, 10, 12, 13))
upsetjs() %>% fromList(listInput) %>% interactiveChart() %>% chartProps(exportButtons=list(share=FALSE, vega=FALSE, dump=FALSE))
```
